test/katex-spec.js

```diff
@@ -2586,9 +2606,8 @@ describe("href and url commands", function() {
 
     it("should allow single-character URLs", () => {
         expect`\href%end`.toParseLike("\\href{%}end");
-        expect`\href %end`.toParseLike("\\href{%}end");
         expect("\\url%end").toParseLike("\\url{%}end");
-        expect("\\url %end").toParseLike("\\url{%}end");
+        expect("\\url%%end\n").toParseLike("\\url{%}");
```

Comment:
Why are multiple `"%"` in a row not allowed in URLs?

Reply:
@kevinbarabash Multiple characters should be enclosed in brackets, in url too.

Reply:
whoops... forgot the basics of TeX syntax 😅